### PR TITLE
Update microsoft-defender-endpoint-linux.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a couple of related articles on onboarding documentation via Azure Arc

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/8413

File: microsoft-365/security/defender-endpoint/microsoft-defender-endpoint-linux.md [hunk 1]
```diff
@@ -223,3 +223,9 @@ High I/O workloads from certain applications can experience performance issues w
 ## Resources
 
 - For more information about logging, uninstalling, or other topics, see [Resources](linux-resources.md).
+  
+## Related articles
+  
+-  [Protect your endpoints with Defender for Cloud's integrated EDR solution: Microsoft Defender for Endpoint](https://docs.microsoft.com/en-us/azure/defender-for-cloud/integration-defender-for-endpoint?tabs=windows)
+-  [Connect your non-Azure machines to Microsoft Defender for Cloud](https://docs.microsoft.com/en-us/azure/defender-for-cloud/quickstart-onboard-machines?pivots=azure-arc)
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/microsoft-defender-endpoint-linux.md]:
```suggestion
-  [Protect your endpoints with Defender for Cloud's integrated EDR solution: Microsoft Defender for Endpoint](/azure/defender-for-cloud/integration-defender-for-endpoint)
-  [Connect your non-Azure machines to Microsoft Defender for Cloud](/azure/defender-for-cloud/quickstart-onboard-machines)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**